pkg/common/common.go

```diff
@@ -48,12 +48,14 @@ const (
 	AlloyEventsLoggerAppNamespace = "kube-system"
 
 	MaxBackoffPeriod = "10m"
-	LokiURLFormat    = "https://%s/loki/api/v1/push"
+	LokiBaseURL      = "https://%s"
+	LokiPushURL      = LokiBaseURL + "/loki/api/v1/push"
```

Comment:
Those are not the urls themselves but the format for it

```suggestion
	LokiBaseURLFormat     = "https://%s"
	LokiPushURLFormat      = LokiBaseURLFormat + "/loki/api/v1/push"
```